### ISSUE TITLE
Media 'Image Editor' button missing

### ISSUE THREAD

**Issue by edikaufmann** [OrchardCMS/Orchard#6234]
When developing my WebSite using VS2015 or WebMatrix I'm able to edit my images:
Admin --> Media --> clicking an image --> Image Editor 

HOWEVER, as soon as I deploy my Site the 'Image Editor' button won't appear anymore.

Following Modules (and its prereq's) are enabled:
Media.Library
Media.Library.Search
Media.Library.Processing
Image.Editor

I'm running 1.9.2; are there any updates needed/available for these relevant modules!?
(I'm NOT on Azure)
Thanks for any hint, ed

**Comment by edikaufmann** [comment]
my first comment isn't true anymore:
I.E. the next morning : the  'Image Editor' button disappeared again!
quote
When developing my WebSite using VS2015 or WebMatrix I'm able to edit my images:
 Admin --> Media --> clicking an image --> Image Editor
end quote

**Comment by edikaufmann** [comment]
well, after testing some more (2 days):
the results on my systems (WebMatrix, VS2015 and the deployed (non-Azure) sites:
the results   seem unpredictable. Sometimes the 'Image Button' appears, the next time it's not
there anymore. (Also, I'm not able to find any hints in the log files or when tracing)
I appreciate any hint you might have. thanks, ed

**Comment by UptownKeith** [comment]
I am having exactly the same problem. I am running on 1.9.1 on IIS on Server 2012

**Comment by edikaufmann** [comment]
As you see I closed that issue due to  strange behaviour on this subject i.e. the behaviour was not predictable anymore - therefore I closed it and did some more 'investigation' --> without success! That's the reason I opened a new one again:  "Media ImageEditor button not showing  #6285" in GitHub AND in the old forum!? Reason for that: PS:

PS: As a GitHub novice I (re)placed  - probably  erroneously - this same issue again on GitHub (#6285)!
I (only) understand that GitHub is firstly an App for distributed revision control using (local) repositories. After Orchard introduced GitHub: where is the correct place to report issues like this? I see reported issues also on GitHub - are all of them based on local repositories contributing to fix Orchard development / design errors?

Which forum is the correct one for issues like this?
Thanks for a clarification on this and the issue itself ? ed
